src/symbols/Node/Node.tsx

```diff
@@ -266,37 +274,49 @@ export const Node: FC<Partial<NodeProps>> = ({
         {...bind()}
         ref={nodeRef}
         tabIndex={-1}
-        onKeyDown={event => {
-          event.preventDefault();
-          if (!isDisabled) {
-            onKeyDown(event, properties);
-          }
-        }}
-        onClick={event => {
-          event.preventDefault();
-          event.stopPropagation();
-          if (!isDisabled && canSelect) {
-            onClick(event, properties);
-          }
-        }}
+        onKeyDown={
+          onKeyDown &&
```

Comment:
Your PR says fix unneeded renders - if this check is the only real material change - I'm curious why this helped ( just personally for me ).

Reply:
I use node hover styles and if there are more then 50 nodes - get getting significant delays with this effect.
Started researching and found out that every onMouseEnter and omMouseLeave elements are re-rendered.

You can reproduce it with any current story on documentation, just start React profiler and move the cursor over the nodes, you will see re-renders